rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+
+It is necessary for the team to have the authority to ban, as it is the basis of all the soft power the team wields. However, the team should prefer to use bans only as a last resort. When somebody is excluded from a community, the community has no further power to change their heart, help them grow, or otherwise influence them. Bans should not be performed vindictively or as punishments, but only as necessary actions for the well being of the remaining community members. A ban is a sad occasion, no matter how necessary it may be.
+
+The team should, whenever possible, lead by example. This may include mediating disputes, either privately or publicly as appropriate. It may also include asking community members to refrain from certain topics of discussion or from certain conflicts. Community members are expected to follow these directions, or to appeal them privately by escalating to the full community team. It is valid and important for community members to give feedback on the community team's actions, but team members' decisions are not to be mocked, flaunted, or relitigated in the court of public opinion.
+
+The above are some suggested remedies, but these should not be taken as prescriptive. The full space of social actions is open to team members as potential tools for resolving disputes. Team members are expected to use their authority only in ways that embody justice and that are consistent with the community's values, as formalized in the code of conduct and statement of values. Team members are expected to recognize the power dynamic that they are part of, and to be proactive and explicit in making sure community members know when they're speaking in an official capacity and when they're speaking in a personal capacity. Team members are not expected to be impartial in the sense of pretending not to have personal beliefs or goals, but are expected to hold the community's values above their own, in their official actions.
+
+The community team will hold its members accountable to these ideals.
+
+## Code of Conduct and Statement of Values
+[values]: #values
+
+Following the adoption of this RFC, the code of conduct and statement of values will be displayed on a dedicated page on the NixOS website. They will read as follows, unless amended by a future RFC.
+
+The Nix community recognizes that its members come from many walks of life, and that it is natural for us to disagree on some things. Nonetheless, we are all committed to certain values, to allow us to work together and build awesome technologies.
+
+Community members are expected to follow these values in their interactions in all official Nix social spaces, including websites, chat rooms, bug threads, and conferences. We have a community team who will set and enforce social norms, to make that easier for everyone. faCommunity members are expected to follow the community team's requests.
```

Comment:
Seems like there is a typo in the last sentence: "faCommunity".

Reply:
Thanks for flagging that. No idea how it slipped through.